packages/discord-argument/package.json

```
@@ -0,0 +1,60 @@
+{
+	"name": "@sapphire/discord-argument",
```

```suggestion
	"name": "@sapphire/argument-parser",
```

Name the package like this. Reflect this everywhere (readme, folder name, etc)